docs/content/reference/latest/assessment-plan/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Assessment Plan Model v1.0.0 Reference"
+title: "Assessment Plan Model v Reference"
```

Comment:
Why this change?

Reply:
This is based in some automation issues with the model documentation script. We discussed this during a PR review meeting with Dave and Rene. I will open a bug for this to make sure it gets tracked later.

Reply:
Ok opened up usnistgov/OSCAL#1106 to track this specifically and move a fix outside of this PR conversation. Hope that clarifies, Brad.